nautobot/utilities/tests/test_forms.py

```diff
@@ -826,23 +820,26 @@ def test_dynamic_filter_form_with_data_and_prefix(self):
         request_querydict.setdefault("created", "2022-09-05")
         request_querydict.setlistdefault("asn", ["4"])
 
+        site_filter_set = SiteFilterSet()
```

Comment:
This batch of changes will be fun to merge into `next`...

Reply:
Yeah, big `:oof:` when I read these tests. We should have used Location or something else. 🤷